docs/DeveloperGuide.md

```diff
@@ -552,6 +553,32 @@ Use case ends.
     * 1b1. AscendNetwork shows 0 person is added.
       Use case resumes at step 1.
 
+**Use case: UC13 - Summarize information about current list of contacts**
```

Comment:
Also I think we agreed to use 'persons' in DG since the class is called 'Person'